src/js/components/Tabs/doc.js

```diff
@@ -48,6 +48,21 @@ currently active tab changes.`,
 };
 
 export const themeDoc = {
+  'global.borderSize': {
+    description: 'The size of the border',
+    type: 'object',
+    defaultValue: undefined,
+  },
+  'global.colors.border': {
+    description: 'color of the border',
+    type: 'string',
```

Comment:
it can actually accept an object of `dark` and `light` as well, see https://codesandbox.io/s/o9xo5p13zz
`type: 'string | { dark: string, light: string }',`